bp_top/syn/Makefile

```diff
@@ -5,51 +5,24 @@ include $(TOP)/Makefile.common
 
 export SYN_PATH     := $(BP_TOP_DIR)/syn
 export TB_PATH      := $(BP_TOP_DIR)/test/tb
-export MEM_PATH     := $(BP_COMMON_DIR)/test/mem
 
 export LOG_PATH     := $(BP_TOP_DIR)/syn/logs
 export RESULTS_PATH := $(BP_TOP_DIR)/syn/results
 export REPORT_PATH  := $(BP_TOP_DIR)/syn/reports
+export TOUCH_PATH   := $(BP_TOP_DIR)/syn/touchfiles
 
-TB          ?= bp_softcore
-CFG         ?= e_bp_softcore_cfg
-
-COH_PROTO   ?= mesi
-CCE_MEM_PATH = $(BP_ME_DIR)/src/asm/roms
-CCE_MEM      = $(COH_PROTO).mem
-
-DRAMSIM_CH_CFG  ?= DDR3_micron_32M_8B_x8_sg25E.ini
-DRAMSIM_SYS_CFG ?= system.ini
+export TB    ?= bp_tethered
+export CFG   ?= e_bp_softcore_cfg
+export SUITE ?= bp_tests
+export PROG  ?= hello_world
+export TAG   ?= none
```

Comment:
What is the intended functionality of TAG?

Reply:
Good question! It's not fully implemented yet.  It lets users tag their tests based on something other than the configuration and program -- for example, which testing corner you're using in a VLSI flow